### PR TITLE
Changed password encryption and decryption error message

### PR DESCRIPTION
**Pull request by zhili1208**
https://github.com/NuGet/Home/issues/2942

### CODE REVIEW

**Comment by joelverhagen** [comment]
Please consider merging to `dev`, so that people upgrading from `3.5.0-beta2` to `3.5.0-rtm` (or whatever) don't see this message regress.

File: src/NuGet.Core/NuGet.Configuration/Resources.resx [hunk 1]
```diff
@@ -151,10 +151,10 @@
     <value>Unexpected failure reading NuGet.Config. Path: '{0}'.</value>
   </data>
   <data name="UnsupportedDecryptPassword" xml:space="preserve">
-    <value>Password decryption failed for source : '{0}' is unsupported on this platform. A clear text password may be used instead.</value>
+    <value>Password decryption is not supported on this platform (Windows/.NET Core). The following feed uses an encrypted password: '{0}'. You can use a clear text password as a workaround, or pass it on the commandline</value>
```
Comment (joelverhagen) [inline src/NuGet.Core/NuGet.Configuration/Resources.resx]: nit: sentence should end in a period (`.`).

**Comment by joelverhagen** [comment]
Address comment and :shipit:

**Comment by alpaix** [comment]
Message is deceiving. Interactive password is not supported on .NET Core. Encrypted passwords are not supported on all platforms not just Windows.

**Pull request merged**